android.py

```diff
@@ -0,0 +1,518 @@
+#!/usr/bin/env python
```

Comment:
Why do we need this python script ?
Also there was an accidentally committed file `__pycache__/android.cpython-37.pyc`